### PR TITLE
docs(pypi): Update Description to remove reference to react frontend.

### LINKED ISSUE #37: [DOC] Update description on  pyproject.toml to not have reference to react app.
** What is missing documentation **

** Where have you tried to look **

** How it can be Improved **

### PR DESCRIPTION
**Pull request by gxldCptRick**
## Describe your changes

Removing bit of the description that referenced a bundled react frontend because I don't wanna make it.

## Issue ticket number and link

Closes #37 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have defined what tests cover my change and or added new test to cover change. 
- [x] I have added comments to the change and updated any relevant docs.
- [x] I have pulled the latest in the main branch and merged them ahead of time with no conflicts.

### CODE REVIEW

**Review by Dean-Pesek — DISMISSED** [review]
gross

File: pyproject.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ build-backend = "pdm.backend"
 [project]
 name = "zombie_nomnom_api"
 dynamic = ["version"]
-description = "An API for the zombie-nomnom pypi package with its own packaged react-app to load up and work with."
+description = "An API for the zombie-nomnom pypi package exposing functionality via a graphql."
```
Comment (vCertxfiedGoat) [inline pyproject.toml]: a graphql?

**Review by vCertxfiedGoat — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (vCertxfiedGoat) [reply pyproject.toml]: g isnt a vowel sir

Comment (gxldCptRick) [reply pyproject.toml]: IS IT BETTER NOW

Comment (vCertxfiedGoat) [reply pyproject.toml]: 🖕

**Review by vCertxfiedGoat — APPROVED** [review]

**Pull request merged**